cirq-google/cirq_google/api/v2/run_context.proto

```diff
@@ -100,6 +109,40 @@ message DeviceParameter {
     // by the sweep values themselves.
 }
 
+// A bundle of multiple DeviceParameters and their values.
+// The main use case is to set those parameters with the
+// values from this bundle before executing a circuit sweep.
+// Note multiple device parameters may have common ancestor paths
+// and/or share the same parameter names. A DeviceParamaetersDiff
+// stores the resource groups hierarchy extracted from the DeviceParameters'
+// paths and maintains a string tables; thereby storing ancestor resource
```

Comment:
Nit: should this be `maintains a table of strings`

Reply:
updated per suggested.